src/style.css

```diff
@@ -1,3 +1,88 @@
+/* stylelint-disable font-family-name-quotes */
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.countainer{
+  width: 1200px;
+  display: flex;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+}
+
+.logo {
+  text-decoration: none;
+  height: 40px;
+  padding: 10px 0;
+
+}
+
+.categories__ul {
+  display: flex;
+  list-style-type: none;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.categories__li {
+  height: 100%;
+  margin-left: 20px;
+}
+
+.categories__li:first-child{
+  margin-left: 0;
+}
+
+.categories__li a {
+  display: flex;
+  align-items: center;
+  height: 100%;
+  text-decoration: none;
+  color: #000;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: "Roboto";
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  cursor: pointer;
+  transition: color 0.3s ease;
+  position: relative;
+  justify-content: center
+}
+
+
+
+.categories__li a:hover {
+  color: #00ACDC;
```

Comment:
make variable for color and use in all places.

Reply:
<img width="653" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/8999f37a-ac9d-450b-bb04-b5b18c64254f">

Reply:
<img width="610" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/f5b39f58-ce2a-421d-9884-3bd9c6d7bb0c">
still not fixed, do you have some problems with that?